ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -80,21 +137,15 @@ export default class ListHooks extends Component {
         {!hookGroups && loading && <Spinner loading />}
         <ErrorPanel fixed error={error} />
         {hookGroups && (
-          <MuiTreeView
-            // key is necessary to expand the list of hook when searching
-            key={hookSearch}
-            defaultExpanded={Boolean(hookSearch)}
-            listItemProps={{ color: classes.listItemProps }}
-            searchTerm={hookSearch || null}
-            softSearch
-            tree={tree}
-            onEmptySearch={
-              <Typography variant="subtitle1">
-                No items for search term {hookSearch}
-              </Typography>
-            }
-            Link={Link}
-          />
+          <TreeView
+            expanded={this.state.expanded}
+            selected={this.state.selected}
+            onNodeToggle={handleToggle}
```

Comment:
You can use destructuring assignment for these two state values in render: `const { expanded, selected } = this.state`

Reply:
done